arcdps_mock/main.cpp

```diff
@@ -161,10 +272,21 @@ int Run(const char* pModulePath, const char* pMockFilePath)
 		// Show options window (mirror of arcdps options)
 		{
 			ImGui::Begin("Options");
+			ImGui::Checkbox("Logs", &combatMock.showLog);
 			if (TEST_MODULE_EXPORTS.options_end != nullptr)
 			{
 				TEST_MODULE_EXPORTS.options_end();
 			}
+
+			if (TEST_MODULE_EXPORTS.options_windows != nullptr) {
+				TEST_MODULE_EXPORTS.options_windows("skills");
+				TEST_MODULE_EXPORTS.options_windows("metrics");
+				TEST_MODULE_EXPORTS.options_windows("dps");
+				TEST_MODULE_EXPORTS.options_windows("log");
+				TEST_MODULE_EXPORTS.options_windows("bufftable");
+				TEST_MODULE_EXPORTS.options_windows("Errors");
+				TEST_MODULE_EXPORTS.options_windows(0);
```

Comment:
nullptr? :)

Reply:
yes, arcdps is calling it with a nullptr at the bottom, so we can have all the checkboxes on the top :)

Reply:
I just meant to use nullptr instead of 0 haha